tests/scripts/thread-cert/node.py

```diff
@@ -4072,6 +4072,35 @@ def get_ip6_address(self, address_type: config.ADDRESS_TYPE):
         except Exception as e:
             return super(LinuxHost, self).get_ip6_address(address_type)
 
+    def get_br_peers(self) -> List[str]:
+        # Example output of `br peers` command:
+        #   rloc16:0xa800 age:00:00:50
+        #   rloc16:0x6800 age:00:00:51
+        #   Done
+        self.send_command('br peers')
+        return self._expect_command_output()
+
+    def get_br_peers_rloc16s(self) -> List[int]:
+        """parse `br peers` output and return the list of RLOC16s"""
+        return [
+            int(pair.split(':')[1], 16)
+            for line in self.get_br_peers()
+            for pair in line.split()
+            if pair.split(':')[0] == 'rloc16'
+        ]
+
+    def get_br_routers(self) -> List[str]:
+        # Example output of `br routers` command:
+        #   fe80:0:0:0:42:acff:fe14:3 (M:0 O:0 Stub:1) ms-since-rx:144160 reachable:yes age:00:17:36 (peer BR)
+        #   fe80:0:0:0:42:acff:fe14:2 (M:0 O:0 Stub:1) ms-since-rx:45179 reachable:yes age:00:17:36
+        #   Done
+        self.send_command('br routers')
+        return self._expect_command_output()
+
+    def get_br_routers_ip_addresses(self) -> List[IPv6Address]:
+        """parse `br routers` output and return the list of IPv6 addresses"""
+        return [IPv6Address(line.split()[0]) for line in self.get_br_routers()]
+
```

Comment:
Suggest moving these function close to other `br` related ones in this file?

We have `def enable_br(self):`, `def disable_br(self):`, `def get_br_omr_prefix(self)`, ...

Reply:
sg, thanks! I will update

Reply:
update!